capstone/src/main/java/com/step902020/capstone/User.java

```diff
@@ -0,0 +1,110 @@
+package com.step902020.capstone;
+
+import org.springframework.cloud.gcp.data.datastore.core.mapping.Entity;
+import org.springframework.cloud.gcp.data.datastore.core.mapping.Field;
+import org.springframework.data.annotation.Id;
+import java.util.*;
+
+@Entity(name = "user")
+public class User {
+  @Id
+  Long datastoreId;
+
+  Long timestamp;
+
+  @Field(name="firstname")
+  String firstName;
+
+  @Field(name="lastname")
+  String lastName;
+
+  @Field(name="email")
+  String email;
+
+  @Field(name="university")
+  String university;
+
+  @Field(name="description")
+  String description;
+
+  @Field(name="image")
+  String image;
+
+// these fields are going to be implemented in the future.
+
+//   @Field(name="saved-events")
+//   HashSet<String> savedEvents;
+
+//   @Field(name="saved-organizations")
+//   HashSet<String> savedOrganizations;
```

Comment:
I agree with Lian, the TODOs are incredibly helpful! But I know the one thing Elena always gets on me for are comments, she may want you to stash them for later and just to keep running code in PRs. Just a heads up!